src/components/SwapForm/service/getBalanceNear.ts

```diff
@@ -0,0 +1,35 @@
+import { formatUnits } from "viem"
+
+import { nearAccount } from "@src/utils/near"
+import {
+  NEAR_MIN_RESERVE_BALANCE,
+  NEAR_STORAGE_COST_PRE_BYTE,
+} from "@src/constants/contracts"
+
+export const minimumNearBalance = (storageUsed: number): number => {
+  const storageCostPerByte = NEAR_STORAGE_COST_PRE_BYTE
```

Comment:
PRE - typo,
Plus maybe we could just use a constant

Reply:
Thanks for the catching the typo. Yeap, we can make it as constant.